generate/types.go

```diff
@@ -65,6 +65,14 @@ type (
 		GraphQLName            string
 		Marshaler, Unmarshaler string
 	}
+	// goTypenameForBuiltinType represents a builtin type that was
+	// given a different name due to a `typename` directive.  We
+	// create a type like `type MyString string` for it.
+	goTypenameForBuiltinType struct {
```

Comment:
I might also name this goTypeForBuiltinType.

Reply:
(I would have called it `goNamedType`, but any of these work!)